src/content/api/loyalty-program-memberships.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Loyalty Program Memberships
+description: Loyalty Program Memberships model and related endpoints
+nav:
+  path: API/Loyalty Programs
+  order: 2
+---
+import Properties from '../../components/Properties.astro';
+import Property from '../../components/Property.astro';
+import Error from '../../components/Error.astro';
+import Endpoint from '../../components/Endpoint.astro';
+import Badge from '../../components/Badge.astro';
+
+A Loyalty Program Membership is a record of a user being onboarded to a Loyalty Program.
+
+## Loyalty Program Membership Model
+
+<Properties>
+  <Property name="id" type="string">
+    The unique identifier of the Loyalty Program Membership.
+  </Property>
+
+  <Property name="accountId" type="string"> 
+    The [Account](/api/accounts/) that is being onboarded to the Loyalty Program.
+  </Property>
+
+  <Property name="programId" type="string"> 
+    The id of the Loyalty Program that the Membership is created for.
+  </Property>
+
+  <Property name="name" type="string">
+    The display name of the Loyalty Program.
+  </Property>
+
+  <Property name="description" type="string">
+    The displayable description for the Loyalty Program.
+  </Property>
+
+  <Property name="status" type="string">
+    The status of the Loyalty Program Membership.
+  </Property>
+
+  <Property name="mediaUploadId" type="string">
+    The id of the [Media Upload](/api/media-uploads/) image of the Loyalty Program.
+  </Property>
+
+  <Property name="img" type="string">
+    The img URL of the Loyalty Program.
+  </Property>
+
+  <Property name="createdAt" type="timestamp">
+    When the Loyalty Program Membership was created.
+  </Property>
+
+  <Property name="createdBy" type="crn">
+    The User or API Key that created the Loyalty Program Membership.
+  </Property>
+
+  <Property name="test" type="boolean">
+    `true` if the Loyalty Program Membership is for testing purposes only.
+  </Property>
+</Properties>
+
+---
+
+<Endpoint
+  path="/api/loyalty-programs/{loyaltyProgramId}/memberships"
+  filename="loyalty-program-memberships-create"
+>
+  ## Create Loyalty Program Memberships <Badge type="experimental"/>
+
+  This endpoint allows you to create [ Loyalty Program Memberships ](#loyalty-program-membership-model) to onboard users to [ Loyalty Programs ](#loyalty-program-model).
+
+  <Properties>
+    <Property name="accountId" type="string" required>
+      The [Account](/api/accounts/) that is being onboarded to the Loyalty Program.
+    </Property>
+
+    <Property name="idempotencyKey" type="string" required>
+      Client-supplied identifier that prevents double creation.
+    </Property>
+  </Properties>
+
+  <Properties heading="Errors">
+    <Error code="403" message="MEMBERSHIP_ALREADY_CREATED">
+     Membership has already been created for this Loyalty Program with this idempotency key.
```

Comment:
I've update the API design to remove the idempotencyKey field - we will be using accountId as the idempotency to prevent multiple memberships per account and loyalty program.
This descriptoin should be updated to `A membership has already been created for this account and this loyalty program`

Reply:
@seliwe  Yep, I've updated the error description. :) 
I noticed in Notion that we've also changed the error name from `MEMBERSHIP_ALREADY_CREATED` to `MEMBERSHIP_ALREADY_EXISTS`.  Are we also changing the error name as well ?